vendor/src/github.com/docker/notary/client/delegations.go

```diff
@@ -0,0 +1,294 @@
+package client
+
+import (
+	"encoding/json"
```

Comment:
Also, this code only deals with changelists, which are not part of the TUF spec, they are completely our own invention/implementation.